diff -Nurp mariadb-10.1.30.orig/cmake/install_layout.cmake mariadb-10.1.30/cmake/install_layout.cmake
--- mariadb-10.1.30.orig/cmake/install_layout.cmake	2017-12-21 09:48:45.000000000 -0600
+++ mariadb-10.1.30/cmake/install_layout.cmake	2018-01-05 11:15:01.694892933 -0600
@@ -162,8 +162,6 @@ SET(INSTALL_MYSQLDATADIR_RPM
 
 SET(INSTALL_UNIX_ADDRDIR_RPM            "${INSTALL_MYSQLDATADIR_RPM}/mysql.sock")
 SET(INSTALL_SYSTEMD_UNITDIR_RPM         "/usr/lib/systemd/system")
-SET(INSTALL_SYSTEMD_SYSUSERSDIR_RPM     "/usr/lib/sysusers.d")
-SET(INSTALL_SYSTEMD_TMPFILESDIR_RPM     "/usr/lib/tmpfiles.d")
 
 #
 # DEB layout
@@ -193,8 +191,6 @@ SET(INSTALL_MYSQLDATADIR_DEB
 
 SET(INSTALL_UNIX_ADDRDIR_DEB            "/var/run/mysqld/mysqld.sock")
 SET(INSTALL_SYSTEMD_UNITDIR_DEB         "/lib/systemd/system")
-SET(INSTALL_SYSTEMD_SYSUSERSDIR_DEB     "/usr/lib/sysusers.d")
-SET(INSTALL_SYSTEMD_TMPFILESDIR_DEB     "/usr/lib/tmpfiles.d")
 
 #
 # SVR4 layout
@@ -236,7 +232,7 @@ SET(OLD_INSTALL_LAYOUT ${INSTALL_LAYOUT}
 # layout is chosen)
 FOREACH(var BIN SBIN LIB MYSQLSHARE SHARE PLUGIN INCLUDE SCRIPT DOC MAN SYSCONF SYSCONF2
     INFO MYSQLTEST SQLBENCH DOCREADME SUPPORTFILES MYSQLDATA UNIX_ADDR
-    SYSTEMD_UNIT SYSTEMD_SYSUSERS SYSTEMD_TMPFILES)
+    SYSTEMD_UNIT)
   SET(INSTALL_${var}DIR  ${INSTALL_${var}DIR_${INSTALL_LAYOUT}}
   CACHE STRING "${var} installation directory" ${FORCE})
   MARK_AS_ADVANCED(INSTALL_${var}DIR)
diff -Nurp mariadb-10.1.30.orig/support-files/CMakeLists.txt mariadb-10.1.30/support-files/CMakeLists.txt
--- mariadb-10.1.30.orig/support-files/CMakeLists.txt	2017-12-21 09:48:56.000000000 -0600
+++ mariadb-10.1.30/support-files/CMakeLists.txt	2018-01-05 11:15:23.804900441 -0600
@@ -104,21 +104,6 @@ IF(UNIX)
             ${CMAKE_CURRENT_BINARY_DIR}/mariadb.service
             DESTINATION ${inst_location}/systemd COMPONENT SupportFiles)
 
-    IF(INSTALL_SYSTEMD_SYSUSERSDIR)
-      CONFIGURE_FILE(sysusers.conf.in
-              ${CMAKE_CURRENT_BINARY_DIR}/sysusers.conf @ONLY)
-      INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/sysusers.conf
-              DESTINATION ${INSTALL_SYSTEMD_SYSUSERSDIR} COMPONENT Server)
-    ENDIF()
-
-    IF(INSTALL_SYSTEMD_TMPFILESDIR)
-      get_filename_component(MYSQL_UNIX_DIR ${MYSQL_UNIX_ADDR} DIRECTORY)
-      CONFIGURE_FILE(tmpfiles.conf.in
-              ${CMAKE_CURRENT_BINARY_DIR}/tmpfiles.conf @ONLY)
-      INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/tmpfiles.conf
-              DESTINATION ${INSTALL_SYSTEMD_TMPFILESDIR} COMPONENT Server)
-    ENDIF()
-
     # @ in directory name broken between CMake version 2.8.12.2 and 3.3
     # http://public.kitware.com/Bug/view.php?id=14782
     IF(NOT CMAKE_VERSION VERSION_LESS 3.3.0 OR NOT RPM)
diff -Nurp mariadb-10.1.30.orig/support-files/sysusers.conf.in mariadb-10.1.30/support-files/sysusers.conf.in
--- mariadb-10.1.30.orig/support-files/sysusers.conf.in	2017-12-21 09:48:56.000000000 -0600
+++ mariadb-10.1.30/support-files/sysusers.conf.in	1969-12-31 18:00:00.000000000 -0600
@@ -1 +0,0 @@
-u @MYSQLD_USER@ - "MariaDB" @MYSQL_DATADIR@
diff -Nurp mariadb-10.1.30.orig/support-files/tmpfiles.conf.in mariadb-10.1.30/support-files/tmpfiles.conf.in
--- mariadb-10.1.30.orig/support-files/tmpfiles.conf.in	2017-12-21 09:48:56.000000000 -0600
+++ mariadb-10.1.30/support-files/tmpfiles.conf.in	1969-12-31 18:00:00.000000000 -0600
@@ -1 +0,0 @@
-d @MYSQL_UNIX_DIR@ 0755 @MYSQLD_USER@ @MYSQLD_USER@ -
